onnxruntime/python/tools/quantization/operators/softmax.py

```diff
@@ -82,29 +72,3 @@ def quantize(self):
         nodes.append(qnode)
         self.quantizer.new_nodes += nodes
         return None
-
```

Comment:
Note: See this comment thread for rationale for removing QDQSoftmax
https://github.com/microsoft/onnxruntime/pull/19925/files#r1526256693